samza-api/src/main/java/org/apache/samza/operators/MessageStream.java

```diff
@@ -282,7 +284,7 @@
    * @param <K> the type of key in the table
    * @param <V> the type of record value in the table
    */
-  <K, V> void sendTo(Table<KV<K, V>> table);
+  <K, V> MessageStream<KV<K, V>> sendTo(Table<KV<K, V>> table);
```

Comment:
same as above. Are there any ordering guarantees between the table writes and message propagation?

Reply:
+1, table writes need to happen before the propagation.

Reply:
It is. I wanted to add that to the java documentation so that users don't have to dig through the code to understand the guarantees of the API